contents/tutorials/when-and-how-to-run-group-targeted-a-b-tests.md

```diff
@@ -0,0 +1,88 @@
+---
+title: When and how to run group-targeted A/B tests
+sidebar: Docs
+showTitle: true
+author: ['lior-neu-ner']
+date: 2023-04-21
+featuredImage: ../images/tutorials/when-and-how-to-run-group-targeted-a-b-tests/user-vs-group-a-b-tests.png
+tags: ['experimentation']
+---
+
+A/B tests are a powerful tool for measuring how product changes impact user behavior. However, sometimes changing how one user interacts with your product will affect how others use it too. For example, a change in the Uber App for Drivers will likely affect riders' experience too. In these scenarios, group-targeted experiments enable us to measure the impact of changes beyond an individual user and across a group of users.
+
+## What are group-targeted experiments
+
+In group-targeted experiments, the changes being tested are applied to an entire group, rather than individual users. This means that everyone within a specific group will experience the same conditions. For example, either the control or test variant. This enables you to measure the impact of any changes on the group as a whole, rather than on individual users within the group.
+
+Groups can be companies, geographic locations, or any other set of users that share common characteristics.
+
+![Screenshot of setting the participant type in an experiment](../images/tutorials/when-and-how-to-run-group-targeted-a-b-tests/user-vs-group-a-b-tests.png)
+
+## When to run group-targeted experiments
+
+When the change in how an individual user uses your product will significantly affect the behavior of other users, or when you want to measure the impact of a change on an entire group of users. This is usually seen in the following types of products:
+
+1. B2B SaaS apps, in order to test how a change will affect how an entire company uses your product.
```

Comment:
```suggestion
1. B2B SaaS apps, to test how a change affects how an entire company uses your product.
```

Reply:
You kind of repeat yourself here too. You could just combine this with this with the introduction. Add the example to the bullet in the first paragraph of this sentence

> 2. You want to measure the impact of a change on an entire group of users.